### PR TITLE
Add with-gatsby example.

### PR DESCRIPTION
**Pull request by hiyangguo**

### CODE REVIEW

File: examples/with-gatsby/README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+<!-- AUTO-GENERATED-CONTENT:START (STARTER) -->
```
Comment (simonguo) [inline examples/with-gatsby/README.md]: readme 的格式统一参考:
https://github.com/rsuite/rsuite/tree/master/examples/with-rtlcss

File: examples/with-gatsby/src/components/seo.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+ *
+ * See: https://www.gatsbyjs.org/docs/use-static-query/
+ */
+
+import React from "react"
+import PropTypes from "prop-types"
+import Helmet from "react-helmet"
+import { useStaticQuery, graphql } from "gatsby"
+
+function SEO({ description, lang, meta, title }) {
```
Comment (simonguo) [inline examples/with-gatsby/src/components/seo.js]: SEO 这个组件名称最好换一个。

Comment (hiyangguo) [reply examples/with-gatsby/src/components/seo.js]: @simonguo https://github.com/gatsbyjs/gatsby-starter-default/blob/master/src/components/seo.js 这是他官方 example 给的组件 还要改么？

Comment (simonguo) [reply examples/with-gatsby/src/components/seo.js]: 组件名叫 Head 或者叫 HeadManager 更合适一些

Comment (hiyangguo) [reply examples/with-gatsby/src/components/seo.js]: 他官方的东西 感觉还是不动比较好一些。example 感觉尽量原汁原味点儿吧。

**Comment by bradcypert** [comment]
Are there any examples of using Rsuite with Gatsby with the Nav components and Dropdowns? Mine work fine when I run `gatsby develop` but with `gatsby build && gatsby serve` they do not.

**Pull request merged**